modules/test/playwright/pages/object-web/object-action/ActionBuilderPage.ts

```diff
@@ -0,0 +1,30 @@
+import { Locator, Page, expect } from "@playwright/test";
+
+export class ActionBuilderPage {
+    readonly inputNotificationsCombo: Locator;
+    readonly inputThenCombo: Locator;
+    readonly optionNotification: Locator;
+    readonly page: Page;
+
+    constructor(page: Page) {
+        this.inputNotificationsCombo = page.frameLocator('iframe').getByRole('combobox').getByText('Select an Option')
+        this.inputThenCombo = page.frameLocator('iframe').getByRole('combobox').getByText('Choose an Action');
+        this.optionNotification = page.frameLocator('iframe').getByRole('option', { name: 'Notification' });
+        this.page = page;
+    }
+
+    async chooseNotificationOption() {
+		await this.inputThenCombo.click();
+        await this.optionNotification.click();
+	}
+
+    async checkNotificationsVisible(notificationsName:string[]) {
```

Comment:
Assertions don't belong to a page object, they belong to the test. A page object is a representation of the page and its functionalities. The action builder page does not provide a way for the user to check if notifications are visible. This is needed and done by the test only, and so it belongs there in the `spec.ts`.